lib/builders.nix

```diff
@@ -7,22 +7,20 @@
   inherit (inputs) self;
 
   perInput = system: flake:
-    lib.optionalAttrs (flake ? formatter.${system}) {
-      formatter = flake.formatter.${system};
-    } // lib.optionalAttrs (flake ? packages.${system}) {
-      packages = flake.packages.${system};
-    };
+    lib.optionalAttrs (flake ? formatter.${system}) {formatter = flake.formatter.${system};}
+    // lib.optionalAttrs (flake ? packages.${system}) {packages = flake.packages.${system};};
```

Comment:
i actually prefer the original formatting here

Reply:
nix formatting is cursed